docs/requirements_for_rtd.txt

```diff
@@ -56,4 +56,5 @@ tabulate~=0.8.5
 tornado<5.0
 tzlocal~=2.0
 upf_to_json~=0.9.2
-wrapt~=1.11.1
\ No newline at end of file
+wrapt~=1.11.1
+git+https://github.com/readthedocs/readthedocs-sphinx-search@master
```

Comment:
I'm a bit worried about this dependency. Can we ask for a tagged release or better yet a PyPI package? Alternatively we could reference a specific REF. A bad release here could break our docs, right?

Reply:
> Can we ask for a tagged release or better yet a PyPI package?

Done https://github.com/readthedocs/readthedocs-sphinx-search/issues/51

> Alternatively we could reference a specific REF.

Yes, that should be no problem.

> A bad release here could break our docs, right?

Right.

Reply:
They've just released a version on PyPI